config/routes.rb

```diff
@@ -35,18 +35,18 @@
   # Jobssearch form override (English and Welsh variants)
   constraints(slug: /(jobsearch|chwilio-am-swydd)/) do
     get "/:slug.json"      => redirect("/api/%{slug}.json")
-    get "/:slug(.:format)" => "root#jobsearch"
+    get "/:slug(.:format)" => "jobsearch#show"
   end
 
   # Transaction finished pages
```

Comment:
Oh, are these some sort of special case of a done page? I wonder if we could clean this up somehow (probably as a different piece of work...)